deploy/bases/app/jobs/install.yaml

```diff
@@ -38,8 +38,9 @@ spec:
         args:
         - -c
         - |
+          set -o errexit
           ./bin/install.sh
-          php bin/magento setup:perf:generate-fixtures setup/performance-toolkit/profiles/ce/small.xml
+          php bin/magento setup:perf:generate-fixtures setup/performance-toolkit/profiles/ce/small.xml || echo "Failed generating some data."
```

Comment:
I think `|| echo` could cause silent failures during data generation. Maybe let's try without it?